File: src/test/java/training/katas/ValidParentheses/ValidParenthesesTest.cs
namespace training.katas.ValidParentheses
{
    using NUnit.Allure.Core;
    using NUnit.Framework;
    using System;

    [AllureNUnit]
    [TestFixture]
    public class SolutionTest
    {
        [Ignore("TODO: no way of currently testing this")]
        [Test]
        public void SampleTest1()
        {
            Assert.That(Parentheses.ValidParentheses("()"), Is.EqualTo(true));
        }

        [Test]
        public void SampleTest2()
        {
            Assert.That(Parentheses.ValidParentheses(")(((("), Is.EqualTo(false));
        }
    }
}

File: src/main/java/training/katas/ValidParentheses/valid_parentheses.test.js
/*
Test.assertEquals(validParentheses("()"), true);
Test.assertEquals(validParentheses("())"), false);
*/

const chai = require("chai");
const assert = chai.assert;
const validParentheses = require("./valid_parentheses").validParentheses;

describe("Valid parentheses", function() {
    it("empty string", function() {
        assert.strictEqual(validParentheses(""), true);
    });
});